pkg/webui/components/button/index.js

```diff
@@ -64,8 +65,10 @@ class Button extends React.PureComponent {
       [style.large]: large,
     })
 
+    const Elem = 'href' in rest ? 'a' : 'button'
```

Comment:
`href` is not in proptypes of the component

Reply:
what if we dont need full page reload? there are such use cases for the `<Link />` component from `react-router`.
Instead of bloating props of the button, i prefer such api for the component:
`<Button />` - regular button
`<Button.Link />` - the `<Link />` from `react-router`
`<Button.AnchorLink />` - the native `<a />` link